chainerrl/agents/categorical_dqn.py

```diff
@@ -133,8 +177,19 @@ def _compute_loss(self, exp_batch, errors_out=None):
         # Minimize the cross entropy
         # y is clipped to avoid log(0)
         eltwise_loss = -t * F.log(F.clip(y, 1e-10, 1.))
-        if self.batch_accumulator == 'sum':
-            loss = F.sum(eltwise_loss)
+
+        if errors_out is not None:
+            del errors_out[:]
+            # The loss per example is the sum of the atom-wise loss
```

Comment:
It could be commented that the prioritization by KL divergence is written in the Rainbow paper.